guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -42,24 +42,49 @@ ifdef::katello,satellite[]
 +
 . Regenerate certificates on your {ProjectServer}:
```

Comment:
Users should not have to regenerate certificates everytime they upgrade a Smart Proxy/Capsule. This was needed for one release where we added the certificate files directly and were dropping the use of the RPMs but this is not a general purpose requirement. I think we can just drop all of this and make upgrades simpler.

@evgeni @ekohl Am I missing something with my logic?

Reply:
Ohh sorry, this is specific to 3.1.